test/changed.sh

```diff
@@ -39,6 +39,56 @@ function cleanup {
 }
 trap cleanup EXIT
 
+# Verify that the semver for the chart was increased
+# This function is also in the circleci test runs. The duplication is to make
+# sure this is checked a second time right before merging the PR.
+semvercompare() {
```

Comment:
Can we put these functions in a common file and then source and then source them for both circle and PR tests?

Reply:
Yeah. It's WIP and I want to get it working first.